README.md

```diff
@@ -47,8 +47,9 @@ import { Chart } from "react-google-charts";
   legendToggle
 />
 ```
+You can also check this [step-by-step tutorial](https://cube.dev/blog/react-google-charts-dashboard/?ref=eco-react-google-charts) that will walk you through the creation of a full-fledged dashboard with this library.
```

Comment:
Could we add this after the Examples item in the `Docs` section right below ?  Having it before might confuse the reader and make them think this is the first thing to read to get started for a simple use-case.

Reply:
Makes sense, sure! Moved as you suggested 👍